Updated the JWT automatic renwal notice for additional clarity (#1885)
nginx/documentation: content/includes/licensing-and-reporting/fcp-renewal-caution.md

- content/solutions/about-subscription-licenses/nginx-plus-licensing-workflows.md
---

{{< call-out "caution" "FCP subscription renewals require manual JWT updates" >}}

> If your subscription was renewed under the Flexible Consumption Program (FCP), automatic JWT renewal isn't supported. In this scenario, updated JWT licenses must be [applied manually](https://docs.nginx.com/solutions/about-subscription-licenses/getting-started/#manually-update-license) at each renewal to ensure that NGINX Plus continues operating as expected.

Reviewer: ```suggestion
If your subscription was renewed under the Flexible Consumption Program (FCP), automatic JWT renewal isn't supported. You must [apply updated JWT licenses manually](https://docs.nginx.com/solutions/about-subscription-licenses/getting-started/#manually-update-license) at each renewal to make sure NGINX Plus continues to work.
```

> Subscription details, including the current subscription type, can be verified on [MyF5](https://my.f5.com).

Reviewer: ```suggestion
Check your subscription details, including subscription type, on [MyF5](https://my.f5.com).
```
